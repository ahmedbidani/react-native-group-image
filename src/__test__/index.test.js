import 'react-native'
import React from 'react'
import GroupImage from '../index'

import renderer from 'react-test-renderer'

test('renders correctly', () => {
  const tree = renderer.create(
    <GroupImage />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
