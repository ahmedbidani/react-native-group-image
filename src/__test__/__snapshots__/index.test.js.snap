// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Image
  source={undefined}
  style={
    Object {
      "borderRadius": 24,
      "flexDirection": "row",
      "height": 48,
      "overflow": "hidden",
      "width": 48,
    }
  }
/>
`;
